pkg/dns/dns.go

```diff
@@ -558,6 +637,9 @@ func getSkyMsg(ip string, port int) (*skymsg.Service, string) {
 //   5. Fourth segment is exactly "svc"
 //   6. The remaining segments match kd.domainPath.
 //   7. And federation must be one of the listed federations in the config.
+//   Note: Because of the above conditions, this method will treat wildcard queries such as
+//   *.mysvc.myns.myfederation.svc.domain.path as non-federation queries.
+//   We can add support for wildcard queries later, if needed.
```

Comment:
Yes, I would like us to support wildcard queries for federated services, as this will be an easy way for clients to discover all shared of a service globally, with a single DNS query.  Pretty cool.  Can wait until post-1.3.